hepstats/utils/fit/__init__.py

```diff
@@ -1,25 +1,30 @@
 from contextlib import ExitStack
-import numbers
 
 
-def eval_pdf(model, x, params):
+def get_value(value):
+
+    if hasattr(value, "numpy"):
```

Comment:
I didn’t know about the use of np.array on zfit objects, it actually makes sense to me now. But now I don’t see where this would fail actually.

Reply:
Yeah you can put pretty much everything in a `np.array(value)`. The only different thing is that `np.array(value)` returns an array with a single value, and `value.numpy()` returns a `float`/`int` but I don't see where problems can happen because of that. 

The `zfit.Parameter` do have a `numpy` method however you have to do `np.array(param.value())` to actually get the value.

Reply:
What exactly do you mean about the zfit Parameters?
It's true tough, you're right, that the composed don't have `numpy` attribute, it's also not in the zfit api (it's because they inherit from TensorFlow).
param.value().numpy() should work though as well.

But yeah, is there a difference between a Python scalar and a numpy array scalar? I don't think so

Reply:
> True, you're right, the composed don't have numpy attribute, it's also not in the zfit api (it's because they inherit from TensorFlow).
param.value().numpy() should work though as well.

```python
def get_value(value):
    if hasattr(value, "numpy"):
        return value.numpy()
    else:
        return value

def get_value_new(value):
    return np.array(value)
```

The difference is that to get the value of the parameter this should be called:

```python
get_value(param) == get_value_new(param.value())
```.

Reply:
Yes, sure. This should be unified within zfit though.

And maybe you want to use `zfit.run`, which does the same as above, even on multiple elements. So that the logic of how to convert elements (framework specific) stays contained in the framework (here zfit).
Or does that break things somehow?

Reply:
This is what I was doing before, but I removed that since zfit > 0.4. I think is fine to call `get_value(param.value)` for now.

Reply:
ok! I would still vote for `zfit.run` to keep a certain consistency, e.g. if the backend in zfit changes, `zfit.run` will change accordingly.

But it's a long future thing, not much to change here anyway.

Reply:
Yeah we will see, the good thing this there is only one place to change the code.